packages/ng/department/service/department-v4.service.ts

```diff
@@ -40,7 +45,7 @@ export class LuDepartmentV4Service {
 				>(`/api/v3/departments/scopedtree?fields=id,name&${[`appInstanceId=${this._appInstanceId}`, `operations=${this._operations.join(',')}`, this._filters.join(',')].filter((f) => !!f).join('&')}`)
```

Comment:
Hijacking this PR, when will this call be migrated to v4 API ?
Why is this PR updating the call without operations ?